packages/datadog-plugin-next/test/pages/api/hello/[name].js

```diff
@@ -0,0 +1,9 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
Is the name of this file (i.e. with the square brackets) as intended?

Reply:
It's intended as this is how Next identifies patterns instead of exact matches.